scripts/shared_env

```diff
@@ -16,7 +16,6 @@
 # setting any other common environment variables
 # It is expected to be 'sourced' while the CWD is the root of the repository
 
-export GO15VENDOREXPERIMENT=1
```

Comment:
Can you also update the header of this file? i.e. `2014-2016` -> `2014-2018`

Reply:
Why we delete this environment variable?